packages/core/src/App/Containers/Modals/crypto-transaction-processing-modal/crypto-transaction-processing-modal.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDevice } from '@deriv-com/ui';
+import { Button, MobileDialog, Modal } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { CryptoTransactionProcessingModalContent } from './crypto-transaction-processing-modal-content';
+import './crypto-transaction-processing-modal.scss';
+
+const CryptoTransactionProcessingModal = observer(() => {
+    const { isDesktop } = useDevice();
+    const { ui } = useStore();
+    const { should_show_crypto_transaction_processing_modal, setShouldShowCryptoTransactionProcessingModal } = ui;
+
+    const onCloseModal = () => {
+        setShouldShowCryptoTransactionProcessingModal(false);
+    };
+
+    return (
+        <React.Fragment>
+            {isDesktop ? (
+                <Modal
+                    is_open={should_show_crypto_transaction_processing_modal}
+                    className='crypto-transaction-processing-modal'
+                    title=' '
```

Comment:
Do we need this space in title?

Reply:
@habib-deriv yes, we need it, because if I don't pass anything to title or pass `''` instead if `' '`, then we will not see the header with cross icon, but we need it in this modal. You can see how it looks without `title=' '` and with

<img width="500" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/6fe9db98-618a-4275-a4d2-f70799f7a3e1">
<img width="466" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/08c2593f-dff6-49a4-bce5-741f0dbf7c22">